### PR TITLE
Creates Temp folder if not exist

### PR DESCRIPTION
**Pull request by TurBoss**
Hello, This fix creates the temporal folder if doesnt exist

Thanks

### CODE REVIEW

File: 7thHeaven.Code/RegistryHelper.cs [hunk 1]
```diff
@@ -435,6 +435,13 @@ public static void BeginTransaction()
 
         public static void CommitTransaction()
         {
+
+            string path = Sys.PathToTempFolder;
+            if (!Directory.Exists(path))
+            {
+                Directory.CreateDirectory(path);
+            }
```
Comment (julianxhokaxhiu) [inline 7thHeaven.Code/RegistryHelper.cs:443]: Can you please remove the `path` variable and just use directly `Sys.PathToTempFolder` instead? Thank you!

**Review by julianxhokaxhiu — CHANGES_REQUESTED** [review]

File: 7thHeaven.Code/RegistryHelper.cs (hunk 1 above)
Comment (TurBoss) [reply 7thHeaven.Code/RegistryHelper.cs:443]: sure, give me a sek

Comment (julianxhokaxhiu) [reply 7thHeaven.Code/RegistryHelper.cs:442]: Thanks :)

**Review by julianxhokaxhiu — APPROVED** [review]

**Comment by julianxhokaxhiu** [comment]
Thank you!

**Pull request merged**